apiml/src/main/java/org/zowe/apiml/controller/ReactivePassTicketController.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.ReactiveSecurityContextHolder;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.zowe.apiml.passticket.PassTicketService;
+import org.zowe.apiml.security.common.token.TokenAuthentication;
+import org.zowe.apiml.ticket.TicketRequest;
+import org.zowe.apiml.ticket.TicketResponse;
+import reactor.core.publisher.Mono;
+
+import java.util.Objects;
+
+import static org.apache.http.HttpStatus.SC_UNAUTHORIZED;
+
+@RestController
+@RequestMapping("/gateway/api/v1/auth")
+@Slf4j
+@RequiredArgsConstructor
+public class ReactivePassTicketController {
+
+    private final PassTicketService passTicketService;
+
+    @PostMapping(value = "/ticket", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Generate a passticket for the user associated with a token.",
+        tags = {"Security"},
+        operationId = "GenerateTicketUsingPOST",
+        description = "Use the `/ticket` API to request a passticket for the user associated with a token.\\n" + //
```

Comment:
The text wasn't copied correctly, see:
- \\n instead of \n\n
- // on the end of line

Please consider using """